### PR TITLE
add schema for forward-request

### PR DESCRIPTION
**Pull request by ameba23**

### CODE REVIEW

**Comment by ameba23** [comment]
@KGibb8 you'll notice i didn't use ssb-invite-schema.  i started out with that, but i had to make so many changes that in the end it was looking pretty ugly and hard to follow, so i started fresh.

so now there is no `body` property, but a `secretOwner` property which will contain the feedId of the person who's secret you are requesting to have forwarded.  

we could have an additional property where you specify whether you are requesting all shards or just the one with a gossip.json attachment.  but i couldn't make up my mind about it so i've left it out for now.

as you suggested, i was gonna add a `branch` attribute to `forward` messages. but i found there already is one!  (we must have been thinking ahead!?)  if you can think of any other properties they might need now there is a requests system lemme know.

**Comment by KGibb8** [comment]
Unified into single branch, see https://github.com/blockades/ssb-dark-crystal-schema/pull/22
